src/aat/resources/features/F-1008 - Submit Event Creation Handle Case Links/F-1008-GetUpdateEventToken.td.json

```diff
@@ -0,0 +1,46 @@
+{
+	"title": "should create an event token for correct inputs",
+	"_guid_": "F-1008-GetUpdateEventToken",
+
+	"productName": "CCD Data Store",
+	"operationName": "Retrieve update token for an existing case",
+
+	"method": "GET",
+	"uri": "/caseworkers/{uid}/jurisdictions/{jid}/case-types/{ctid}/cases/{cid}/event-triggers/{etid}/token",
```

Comment:
If a **common TD file** to cover the basics of the API signature does not exist can we create one and use it as this endpoint is being used multiple times in these tests:

* F-1008-GetUpdateEventToken

(you may wish to add a new generic common base TD and reference it in `Befta_Jurisdiction2_Token_For_Update_Case`)


and see also the following in hmcts/ccd-data-store-api#1719:

* F-1005-CreateEventTokenErrorResponse
* S-1006_GetUpdateSuspendedCaseTokenCaseworker
* S-1006_GetUpdateTokenCaseworker
* S-1007_GetUpdateSuspendedCaseToken
* S-1007_GetUpdateToken

* F-1008-GetLinkedCaseUpdateEventToken
* F-1008-GetMultipleLinkedCaseUpdateEventToken
* F-1008-GetUpdateEventToken

Reply:
Done [c129a9a](https://github.com/hmcts/ccd-data-store-api/pull/1872/commits/c129a9a6a6fdff4226e65e0b027691d3cd49352f)